README.md

```diff
@@ -1,8 +1,10 @@
 ## OpenShift 4.3 Designs
 Access design documentation for features slotted in the 4.3 release of OpenShift. Click links below for specific section details.
 
+- [Set Default Storage Class](http://openshift.github.io/openshift-origin-design/web-console/Storage/set-default-storage-class/set-default-storage-class)
 - [Diagnostics](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/diagnostics/diagnostics)
```

Comment:
@lizsurette would we want to add a section for the Diagnostics link to live under if we are adding one for OCS?

Reply:
@matthewcarleton - I'm not against it! I am just hoping to replicate what we have for the 4.2 stuff for now. Maybe it's worth a larger discussion with @beanh66.

Reply:
@matthewcarleton and @lizsurette, I am not sure about this conversation... I believe you are discussing  _ "the placement of the files under an appropriate section"_  - am I right
Please let me know if I need to change anything from my end.

**[NOTE: Currently, I am focusing on DevTool 4.3 deliverables so my response may be delayed]**

Reply:
It's really just about adding a sub-heading under the heading `OpenShift 4.3 Designs` 
so you could add `OpenShift Container Storage` above your link to your designs. We can also add one for Diagnostics, maybe reach out to @beanh66 about what that should be.

Reply:
@matthewcarleton,
Thanks for your comment. I will update the changes as soon as I complete my current story.